ironfish-cli/src/commands/miners/mined.ts

```diff
@@ -46,6 +46,14 @@ export class MinedCommand extends IronfishCommand {
 
     this.log('Scanning for mined blocks...')
 
+    //Truncate fractions from parameters
+    args.stop = MathUtils.floor(args.stop, 0)
+    args.start = MathUtils.floor(args.start, 0)
+
+    //Fix parameters so 1 <= start <= stop
+    args.start = Math.max(args.start, 1)
+    args.stop = Math.max(args.stop, args.start)
```

Comment:
I think these 2 lines should be deleted, as they break logic we explicitly support. I read your ticket and I think there is a misunderstanding in how we treat negative numbers in all of our commands. Can you read the comment I wrote on your ticket, https://github.com/iron-fish/ironfish/issues/830#issuecomment-1048250105?

Reply:
Agreed with the reasoning from the discord conversation (negative number use case). Will remove these lines.